client/connection.go

```diff
@@ -159,7 +165,7 @@ func ConnectSimpleStreams(url string, args *ConnectionArgs) (ImageServer, error)
 }
 
 // Internal function called by ConnectLXD and ConnectPublicLXD
-func httpsLXD(url string, args *ConnectionArgs) (ContainerServer, error) {
+func httpsLXD(host string, args *ConnectionArgs) (ContainerServer, error) {
```

Comment:
We've traditionally been using `uri` when the `url` package is used to avoid masking it.